apps/docs/pages/guides/functions/regional-invocation.mdx

```diff
@@ -0,0 +1,68 @@
+import Layout from '~/layouts/DefaultGuideLayout'
+
+export const meta = {
+  id: 'function-regional-invocation',
+  title: 'Regional Invocation',
+  description: 'How to execute an Edge Functions in a particular region.',
+}
+
+By default, Edge Functions are executed in the region closest to the user making the request. In most cases, this helps to reduce network latency and provide faster responses to the user.
+
+However, if your function does lot of DB operations (eg: adding a bunch of records) or storage operations (eg: uploading files), invoking the function in the same region as your DB may provide better performance.
```

Comment:
```suggestion
However, if your function does lot of database or storage operations, invoking the function in the same region as your DB may provide better performance.

Some situations where this might be helpful include:

- Bulk adding and editing records in your database
- Uploading files
```

I'm writing for clippy here. We should answer questions like "when would I want to use regional deployments"